### PR TITLE
AIs can no longer gain ion laws compelling them to purge all the homosexuals.

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

image

Yeets HOMOSEXUAL and HETEROSEXUAL out of the list of adjectives for AI ION laws.

## Why It's Good For The Game

Because it's going to trigger all the bigots to come out of their rotten woodworks defending why it's totally okay for the AI to get ion laws declaring it purge all the gays (or imprison all the gays or do anything to all the gays) because it's rare.

Seriously though. That the AI has a **chance** to roll ion laws like that just doesn't sit right with me.

HETEROSEXUAL has been removed for the same reason. Sexual orientation should in zero way factor into the AI's laws or law making decisions in any way, shape or form as far as randomly generated ion laws are concerned.

### CODE REVIEW

**Review by Mothblocks — APPROVED** [review]
wtf lol

**Comment by optimumtact** [comment]
there are a lot more verbs that have potentially offensive combinations as well.

I am going to deny this unless you can give a compelling reason.

**Comment by Timberpoes** [comment]
This should be an absolutely non-controvertial removal. It adds nothing to the game at best and actively detracts from the game at worst.

I really do not care enough to debate why the following should not happen by random chance:

https://tgstation13.org/parsed-logs/manuel/data/logs/2020/09/29/round-147321/game.txt
```
[2020-09-29 21:21:32.447] GAME: Ion storm changed laws of *no key*/(Err) to <SNIP> !&*@*: <font color='#547DFE'>YOU MUST HARM HOMOSEXUAL THINGS AND NOT ALLOW HOMOSEXUAL THINGS, THROUGH INACTION, TO ESCAPE HARM</font>, </SNIP>
```

The facts should speak for themselves without me needing to justify it in any way, shape or form.

If you feel that this is somehow, for some reason, beneficial to the game and that keeping this particular feature in the game is of a net benefit to the game as a whole compared to removing it, then feel free to close this PR.

**Comment by Timberpoes** [comment]
As one last comment
> your sexual preference can never be expressed in game, and therefore isn't a valid target that someone could discriminate against you upon
Your sexual preference can be expressed in game, through RP or just through your character in general. Just because there isn't an LGTBAZERTY+-?! drop-down box to select your preferred orientation doesn't mean that every single player is RPing an asexual.

**Comment by notbanevading** [comment]
>  Your sexual preference can be expressed in game, through RP or just through your character in general. 

How? “Hello! I’m gay” isn’t something that ever happens or should even happen in the first place

This is funny anyways

**Comment by notbanevading** [comment]
“Boo hoo AI laws say that gay people should die”

Show some fucking spine. 
(Jannies I’m cool with this being deleted but My previous comment, I think has a good point or two)

**Comment by Jared-Fogle** [comment]
very interesting thoughts notbanevading Joined 22 minutes ago

**Comment by Winterous** [comment]
Honestly I'm surprised at how FEW potentially offensive words are in the list. If anything we should add some. It's cosmic ray electronic interference, shit's as close to RNG as physics GETS.

I will say that homo and hetero do seem a little ill-fitting with the rest of the stuff, but in my view ideally half the goddamn dictionary would be in this kind of thing.

You could also get a law to exterminate "dark humans", probably, but we aren't gonna remove that.
And hell "gas the lizards" is in there, and THAT could be twisted as referring to jews being lizard people for the particularly nutty individual, and isn't it straight up against the rules to have any nazi references in the game?

I think the absolute best argument you could make for removing this is that it might encourage behaviour that would violate that new bigotry rule.

**Comment by msgerbs** [comment]
Absolutely zero surprises in the list of names silently thumbs-downing this lmao

**Comment by msgerbs** [comment]
> there are a lot more verbs that have potentially offensive combinations as well (OBESE) etc

Of all the things you could pick to be a "potentially offensive combination"... obese? What? Also it's not about what's offensive. The fact of the matter is these 2 words seem out of place at best in the list of adjectives, and they're the only ones that are actively discriminatory with just a single word.

Hell, if you want a good reason to remove it, here: Removing these leaves more room for the ion law to have an adjective that the AI can actually see, leaving the ion law more likely to result in actual interesting interaction.

> for the sole fact that your sexual preference can never be expressed in game, and therefore isn't a valid target that someone could discriminate against you upon.

Isn't just more reason not to have this adjective in the list? Ignoring the false idea that something can only be discriminatory if it's a property that can be expressed in game by a specific person (huh??)

**Comment by Winterous** [comment]
> they're the only ones that are actively discriminatory with just a single word
How is that the case?
If you mean that they discriminate in the sense that they pick people out, then that's outright untrue as there's plenty of words that describe traits, like human, lizard, obese, black, etc..
If you mean that they discriminate in the sense of "x-phobia / ism", then how does merely identifying a trait that people have carry any sort of negative weight with just that single word? If I go up to an obviously gay person and say "You are homosexual.", why should they be offended?

**Comment by msgerbs** [comment]
> 
> 
> > they're the only ones that are actively discriminatory with just a single word
> 
> How is that the case?
> If you mean that they discriminate in the sense that they pick people out, then that's outright untrue as there's plenty of words that describe traits, like human, lizard, obese, black, etc..
> If you mean that they discriminate in the sense of "x-phobia / ism", then how does merely identifying a trait that people have carry any sort of negative weight with just that single word? If I go up to an obviously gay person and say "You are homosexual.", why should they be offended?

You're right, that's poor wording. Let me rephrase: Those are the only words in the adjective list that refer directly to a protected class. That's not inherently discriminatory, but as we see here it clearly can become such in combination with some of the other words. It's out of place compared to all the other adjectives.

**Comment by Winterous** [comment]
> You're right, that's poor wording. Let me rephrase: Those are the only words in the adjective list that refer directly to a protected class. That's not inherently discriminatory, but as we see here it clearly can become such in combination with some of the other words. It's out of place compared to all the other adjectives.

Fair.
I agree that they really don't fit in with the rest, but I think that the solution to that is to add more things, fat, skinny, stupid, retarded, genius, smart, slow, fast, crippled, amputee, disabled, deranged, malformed, mutant, clumsy, coordinated, unwell, ill, healthy, well, etc..

Of course since this is the space age and no one cares about country origin anymore it would be strange to add things like African, Asian, etc.. (even though Americanism is on the list but that's more of a cultural gag)... Although, there's quite a lot of things pertaining to Russia. Russia, Soviet Russia, Be Russian, Being Russian, Russians...

Maybe we should add a bunch of nationalities as well? Those are horrendously out of place, even moreso than homo/hetero. Even if it's just continental adjectives and not countries, like European but not German. Because let's be honest Russia is BASICALLY a continent. Hell, why not planetary as well? Martian, Venutian, Jupiterian, Plutonion, Mercurial, Terran, etc..

If we broaden the net, then any perceived 'discriminatory content' is even more clearly just an act of freak random chance, rather than something specifically intended to be provocative or offensive. The solution to potentially discriminatory content is to make it non-discriminatory by virtue of inclusion, not deletion.

**Comment by optimumtact** [comment]
Well if you're not willing to provide any further justification I don't see any point in continuing this process.

**Comment by bloons3** [comment]
LMAO
image
